src/storage-types/src/sources/load_generator.rs

```diff
@@ -71,6 +71,9 @@ impl SourceConnection for LoadGeneratorSourceConnection {
     fn value_desc(&self) -> RelationDesc {
         match &self.load_generator {
             LoadGenerator::Auction => RelationDesc::empty(),
+            LoadGenerator::Clock => {
+                RelationDesc::empty().with_column("time", ScalarType::MzTimestamp.nullable(false))
```

Comment:
Porting over a pending comment from a review on an earlier revision that I meant to submit several days ago:

> What's the rationale behind using `mz_timestamp` as the type here? This is fundamentally wall clock time, and so using `timestamptz` here feels more appropriate to me.